crm/ebay/setup.rst

```diff
@@ -0,0 +1,28 @@
+==============================
+How to configure eBay in Odoo?
+==============================
+
+Create eBay tokens
+==================
+
+In order to create your tokens, you need to create a developer account `there <https://go.developer.ebay.com/>`_.
```

Comment:
instead of "there" or "here" link, you should use one that describe the page you are going to (e.g. "developer ebay website")